app/upgrade/upgrade-execute.php

```diff
@@ -24,7 +24,7 @@
 # try to upgrade database
 if($Upgrade->upgrade_database()===true) {
 	# print success
-	$Result->show("success", _("Database upgraded successfully! <a class='btn btn-sm btn-default' href='".create_link('dashboard')."'>Dashboard</a>"), false);
+	$Result->show("success", _("Database upgraded successfully!")." <a class='btn btn-sm btn-default' href='".create_link('dashboard')."'>"._("Dashboard")."</a>"), false);
```

Comment:
php syntax error. Too many closing brackets ")" in line.

```php
."'>"._("Dashboard")."</a>", false);
```

Reply:
Hmm, why I haven't seen this by myself? Fixed by latest force push.